lewiskyron/Yank
# ENG-TASK-83: Privacy Policy Page

## Description
With the intent of deploying the web extension, I have created a Privacy Policy Page in line with the step highlighted [here](https://github.com/orgs/supabase/discussions/2925) outlining how to remove the `funky supabase URL` that appears when users authenticate with google. 

## Type of Change


## Testing Instructions
1. Open the preview link. 
2. Paste it into an incognito window 
3. and change the route to Private Policy link.
4. Check if the page opens without authentication. 

## Screenshots (if applicable)
<img width="1440" alt="Screenshot 2024-12-27 at 1 24 21 PM" src="https://github.com/user-attachments/assets/1eba59eb-d495-468b-9b34-3e2d8c99a370" />
